docs/INSTALL.md

```diff
@@ -0,0 +1,70 @@
+# Installation
+
+Reports-server has multiple methods for installation: YAML manifest and Helm Chart.YAML manifest is the recommended method to install reports server when kyverno or policy reports CRDs are already installed in the cluster. Helm chart is the most flexible method as it offers a wide range of configurations.
+
+If kyverno is already installed in the cluster, follow the [migration guide](#migration).
+
+Reports-server comes with a postgreSQL database. It is recommended to bring-your-own postgres database to have finer control of database configurations ([see database configuration guide](#database-configuration)).
+
+### YAML Manifest
+It is recommended to install Reports-server using `kubectl apply`, especially when you have policy reports CRDs installed in your cluster ([see migration guide](#migration)). To install reports server using YAML manifest, run the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kyverno/reports-server/main/config/install.yaml
+```
+
+### Helm Chart
+
+Reports-server can be deployed via a Helm chart for a production install–which is accessible either through the reports-server repository.
+
+In order to install reports-server with Helm, first add the Reports-server Helm repository:
+```bash
+helm repo add reports-server https://kyverno.github.io/reports-server
+```
+
+Scan the new repository for charts:
+```bash
+helm repo update
+```
+
+Optionally, show all available chart versions for reports-server.
+
+```bash
+helm search repo reports-server --l
+```
+
+You can install reports server in any namespace. This example uses `reports-server` as the namespace:
+
+```bash
+kubectl create namespace reports-server
+```
+
+Install the reports-server chart:
+```bash
+helm install reports-server --namespace reports-server reports-server/reports-server --devel
```

Comment:
`--devel` is for pre-release only.

Reply:
yes but I could not install the chart without the flag at the moment

Reply:
Yes because we are on alpha versions. For GA release the user doesn't need this flag.

Reply:
@vishal-chdhry ?

Reply:
@realshuting updated

Reply:
thx!